k8s/helm/yona-server/requirements.yaml

```diff
@@ -4,5 +4,5 @@ dependencies:
     repository:  "https://kubernetes-charts.storage.googleapis.com"
   - name: ldap
     version: 0.1.0
-    repository:  "file://../ldap"
+    repository:  "https://yonadev.github.io/helm-charts"
```

Comment:
I've merged your PR into the master, synced this one into it and resolved the conflicts. I want to do two more things:
* Rename yona-server into yona
* Test it

But before that you can review the current change set already.